examples/weather_report/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+# You must change these to your own details.
+[package]
+name = "rust-webassembly-weather-reports"
+description = "Weather Information System- Get the mood of your city on one click using Rust and Webassembly"
+version = "0.1.1"
+authors = ["Ayush <ayushmishra2005@gmail.com>"]
+categories = ["wasm"]
+readme = "README.md"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.release]
+# This makes the compiled code faster and smaller, but it makes compiling slower,
+# so it's only enabled in release mode.
+lto = true
+
+[features]
+# If you uncomment this line, it will enable `wee_alloc`:
+#default = ["wee_alloc"]
```

Comment:
It's ok to remove this since I think if uncommented this manifest wouldn't work?